apps/articles/client/team_channel.coffee

```diff
@@ -26,18 +31,38 @@ module.exports.TeamChannelView = class TeamChannelView extends Backbone.View
 
   renderFeatured: ->
     $el = $('.team-channel-featured')
-    initCarousel $el,
+    @carousel = initCarousel $el,
       advanceBy: 2
       wrapAround: true
+    @windowResized()
 
   setupStickyNav: ->
-    $('.team-channel-header').waypoint (direction) ->
+    $('.team-channel-header').waypoint (direction) =>
       if direction is 'down'
-        $('.team-channel-nav').addClass 'is-sticky'
+        @$body.addClass 'is-sticky'
       else
-        $('.team-channel-nav').removeClass 'is-sticky'
+        @$body.removeClass 'is-sticky'
     , { offset: -400 }
 
+  windowResized: =>
+    # Advance by 1 for smaller screens
+    if window.matchMedia('(max-width: 900px)').matches
+      @$body.addClass 'is-small'
+      @carousel.navigation.advanceBy = 1
+    else
+      @$body.removeClass 'is-small'
+      @carousel.navigation.advanceBy = 2
+
+  toggleHamburgerNav: ->
```

Comment:
Interesting, will this conflict with whatever view handles the hamburger nav now?